### PR TITLE
httpserver.nim: adding 'Content-Type: application/json' header to the response

### PR DESCRIPTION
**Pull request by Ivansete-status**
# Summary
This PR aims for applying the header "Content-Type: application/json; charset=utf-8" to all the json-rpc responses.

# Issue
https://github.com/waku-org/nwaku/issues/1783

### CODE REVIEW

**Comment by Ivansete-status** [comment]
All tests run passed locally.

**Review by jakubgs — APPROVED** [review]
Thanks.

File: json_rpc/servers/httpserver.nim [hunk 1]
```diff
@@ -41,14 +41,19 @@ proc processClientRpc(rpcServer: RpcHttpServer): HttpProcessCallback =
 
       let body = await request.getBody()
 
+      var headers = HttpTable.init()
+      headers.add("Content-Type", "application/json; charset=utf-8")
```
Comment (kdeme) [inline json_rpc/servers/httpserver.nim:45]: nitpick, I think you can go without the var here:

```suggestion
     let headers = HttpTable.init([("Content-Type", "application/json; charset=utf-8")])
```

**Review by kdeme — APPROVED** [review]

File: json_rpc/servers/httpserver.nim (hunk 1 above)
Comment (Ivansete-status) [reply json_rpc/servers/httpserver.nim]: I've applied the change. Thanks @kdeme !

**Pull request merged**